liquibase-core/src/main/java/liquibase/integration/commandline/Main.java

```diff
@@ -191,6 +197,20 @@ public static int run(String[] args) throws LiquibaseException {
                 return 0;
             }
 
+            //
+            // Look for characters which cannot be handled
+            //
+            for (int i=0; i < args.length; i++) {
+                int index = checkArg(args[i]);
+                if (index >= 0) {
+                    String message =
+                        "Invalid, non-printable character detected in command line at position " +
```

Comment:
Should mention the code point in the message for easier debugging. "Invalid, non printable character #${index} detected in command line..."

Reply:
Oops - just realized the return value is the index where the character is found, not which character was found. Could change the logic of checkArg to return a struct with both values - what was found and where.